cmds/getty/getty.go

```diff
@@ -0,0 +1,205 @@
+// Copyright 2019 the u-root Authors. All rights reserved
```

Comment:
if you can move it to cmds/exp/getty I can approve

Reply:
moved ;)